tests/expose_test.go

```diff
@@ -73,6 +76,35 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 		return job
 	}
 
+	cleanupService := func(serviceName string, namespace string) error {
+		err := virtClient.CoreV1().Services(namespace).Delete(serviceName, &k8smetav1.DeleteOptions{})
+		err = wait.PollImmediate(2*time.Minute, time.Second, func() (done bool, err error) {
+			_, err = virtClient.CoreV1().Services(namespace).Get(serviceName, k8smetav1.GetOptions{})
+			return err != nil && errors.IsNotFound(err), nil
```

Comment:
Why do we need to wait for the service to be removed? Isn't it possible to add a random string to the service name?

Reply:
We can.

Waiting for it to be gone is easy, and takes a very short amount of time.

I find it simpler, and easier to know before-hand what is the name of the service when debugging; it saves me the operation of going to check the real service name.

If you insist I can do it like you suggest.

Reply:
The prefix of the service name will still be the same, so for the purpose of debugging I don't see a big difference. Maybe it will be even easier to read the logs since different test will have different suffix for the service name.